appveyor.yml

```diff
@@ -0,0 +1,13 @@
+version: 1.5.{build}
```

Comment:
does this give folks better visibility into CI failures than Azure Devops? I currently have Devops configured but am not sure what it looks like when folks aren't admins on my instance - if this is better for the community, happy to switch

Reply:
I have no idea about the Azure build pipeline to be honest. I just ended up using AppVeyor for all my bigger projects, I was adding the `.yml` to benefit from features like having the built NuGet instantly available for testing via their own project-specific nuget stream etc.

I didn't want to force a switch but thought it might be interesting to have a working AppVeyor config at hand. [You can see all the build details here](https://ci.appveyor.com/project/nefarius/vcpkg-nuget).